Makefile

```diff
@@ -90,9 +90,6 @@ update_country_dialing_codes:
 check_asset_strings:
 	find ./app/javascript -name "*.js*" | xargs ./scripts/check-assets
 
-generate_deploy_checklist:
-	ruby lib/release_management/generate_deploy_checklist.rb
```

Comment:
this is not part of our current process: https://handbook.login.gov/articles/appdev-deploy.html